test_infra/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.swp
```

Comment:
Do we really need 2 .gitignore? A separate repo is not created for the CDK test infra so perhaps we can just use the root .gitignore?

Reply:
Yep, agree that is better.